src/tiledb/cloud/invites.py

```diff
@@ -0,0 +1,165 @@
+from typing import List
+
+from tiledb.cloud import client
+from tiledb.cloud import rest_api
+from tiledb.cloud import tiledb_cloud_error
+from tiledb.cloud._common import utils
+from tiledb.cloud.rest_api.models.invitation_organization_join_email import (
+    InvitationOrganizationJoinEmail,
+)
+from tiledb.cloud.rest_api.models.organization_roles import OrganizationRoles
+
+
+def accept_invitation(invitation_id: str) -> None:
+    """
+    Accept an invitation.
+
+    :param str invitation_id: the ID of invitation about to be accepted.
+    :return: None
+    """
+    invitation_api = client.build(rest_api.InvitationApi)
+    try:
+        return invitation_api.accept_invitation(invitation=invitation_id)
+    except rest_api.ApiException as exc:
+        raise tiledb_cloud_error.check_exc(exc)
+
+
+def fetch_invitations(**filters):
+    """
+    Fetches a paginated list of invitations.
+
+    :param str organization: name or ID of organization to filter
+    :param str array: name/uri of array that is url-encoded to filter
+    :param str group: name or ID of group to filter
+    :param int start: start time for tasks to filter by
+    :param int end: end time for tasks to filter by
+    :param int page: pagination offset
+    :param int per_page: pagination limit
+    :param str type: invitation type, \"ARRAY_SHARE\", \"JOIN_ORGANIZATION\"
+    :param str status: Filter to only return \"PENDING\", \"ACCEPTED\"
+    :param str orderby: sort by which field valid values include
+                        timestamp, array_name, organization_name
+    :return dict: Invitations and pagination metadata.
+    """
+    invitation_api = client.build(rest_api.InvitationApi)
+    try:
+        return invitation_api.fetch_invitations(**filters)
+    except rest_api.ApiException as exc:
+        raise tiledb_cloud_error.check_exc(exc)
+
+
+def invite_to_organization(
+    organization: str, *, recipients: List[str], role: OrganizationRoles
```

Comment:
`OrganizationRoles` unfortunately isn’t an actual type that you can instantiate (like an `enum.Enum`), it’s more just a container for strings, so it should be annotated as `str`. Also, `recipients` could be more or less any `Sequence[str]`, not just a `List`.